R/drive_upload.R

```diff
@@ -176,14 +176,17 @@ build_drive_upload <- function(input = NULL,
     id <- proc_res$id
   }
 
-  list(
-    method = "PATCH",
-    endpoint =  paste0(id, "?uploadType=media"),
-    path = "upload/drive/v3/files",
+  build_request(
+    path = "https://www.googleapis.com/upload/drive/v3/files/{fileId}",
```

Comment:
I'm still getting my bearings, so humor me. Overall the changes look good. But here I'd expect `https://www.googleapis.com` to be coming from a built-in package default and `path` would hold just `/upload/drive/v3/files/{fileId}`.